### PR TITLE
[B2BP-426] Create react-components package and migrate EC inside B2BP

### PR DESCRIPTION
**Pull request by Meolocious**
This PR refers to Option 2 from: [https://pagopa.atlassian.net/wiki/spaces/SV/pages/983531952/SV-021+Implementazione+componenti+EC+all+interno+di+B2BP](url)

Create the brand new `react-components` package that will contain 🥁.... React components.

#### List of Changes
- Create the `react-components` package
- Install `@pagopa/pagopa-editorial-components` to the `react-components` package
- Add the `Accordion` component (and also the `AccordionItem` component)
- Replace the `Accordion` component within `nextjs-website` application

#### Motivation and Context
We are having troubles with the `@pagopa/pagopa-editorial-components` library, so we are creating our own `react-components` package, which will contain the components coming from `@pagopa/pagopa-editorial-components` library.

#### Types of changes
- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] Doesn't need any documentation updates.

### CODE REVIEW

File: packages/react-components/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "react-components",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "clean": "shx rm -rf dist/",
+    "precompile": "npm run clean",
+    "compile": "tsc",
+    "watch": "tsc --watch",
```
Comment (datalek) [inline packages/react-components/package.json:9]: What about renaming `watch` to `dev`? This way, the developer only needs to execute `npm run dev` in the root.

```suggestion
    "dev": "tsc --watch",
```

Comment (Meolocious) [reply packages/react-components/package.json:9]: I didn't thought about it, I'll change to dev

File: packages/react-components/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "clean": "shx rm -rf dist/",
+    "precompile": "npm run clean",
+    "compile": "tsc",
+    "dev": "tsc --watch",
+    "lint": "eslint src"
+  },
+  "devDependencies": {
+    "@mui/icons-material": "^5.15.10",
+    "@mui/material": "^5.15.10",
+    "@pagopa/pagopa-editorial-components": "^3.1.2",
```
Comment (datalek) [inline packages/react-components/package.json:15]: The `"@pagopa/pagopa-editorial-components": "^3.1.2",` is not used. What do you think about removing it?

```suggestion
```

**Review by datalek — CHANGES_REQUESTED** [review]

**Comment by Meolocious** [comment]
PR abbandoned due to implementing block. Followed the Option 3 implementation from: [https://pagopa.atlassian.net/wiki/spaces/SV/pages/983531952/SV-021+Implementazione+componenti+EC+all+interno+di+B2BP#Opzione-3%3A-Integrazione-EC-su-B2BP-all%E2%80%99interno-della-cartella-Next.JS](https://github.com/pagopa/b2b-portals/pull/url)

Starting PR implementation: [https://github.com/pagopa/b2b-portals/pull/227](url)

**Pull request closed without merging**